amr-wind/convection/incflo_godunov_ppm.H

```diff
@@ -18,17 +18,20 @@ AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE void Godunov_trans_xbc(
     const amrex::Array4<const amrex::Real>& s,
     amrex::Real& lo,
     amrex::Real& hi,
-    amrex::Real& /* uedge */,
+    amrex::Real& velm,
+    amrex::Real& velp,
```

Comment:
I don't understand the use of velm and velp as separate arguments when they are always passed in as the same value in incflo_godunov_advection.cpp . Will that become clear later?

Reply:
I think I did this because some weno-related functions needed to pass different values. Let me check with Ann if we need this and also to confirm if the rest of the code in this file is consistent with how she handles things in Hydro as per the latest changes.

Reply:
I changed it back to not having separate arguments in https://github.com/Exawind/amr-wind/pull/1066/commits/db9143fc47981e9db6522ac62354721007dad7be
which depends on #1199.

Reply:
no longer relevant as this code is being moved to Hydro